tfjs-layers/src/layers/nlp/tokenizers.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ * =============================================================================
+ */
+
+/**
+ *  Tokenizer layers.
+ */
```

Comment:
You could use the license as https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/src/kernels/MaxPoolWithArgmax.ts#L1-L16.

Reply:
done